community-expectations.md

```diff
@@ -0,0 +1,83 @@
+## Community Expectations
```

Comment:
I'm going to wait until I get LGTM on the doc in order to preserve comment history, then I'll move it...